vaccine_feed_ingest/runners/ak/clinic_list/normalize.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python
+
+import datetime
+import os
+import pathlib
+import sys
+from typing import List, Optional
+
+import orjson
+from vaccine_feed_ingest_schema import location as schema
+
+from vaccine_feed_ingest.utils.log import getLogger
+from vaccine_feed_ingest.utils.normalize import normalize_phone, normalize_url
+
+SITE_NAME = "clinic_list"
+RUNNER = "ak"
+
+
+logger = getLogger(__file__)
+
+
+def _get_id(site: dict) -> str:
+    return site["attributes"]["globalid"]
+
+
+def _get_contacts(site: dict) -> Optional[List[schema.Contact]]:
+    ret = []
+    if phone := site["attributes"]["phone"]:
```

Comment:
Are you sure the fields always exist? If not then use `.get` `site["attributes"].get("phone")`

Reply:
It exists in 146/146 of the existing records and sometimes is null; I assume that that's sufficient proof that it's being enforced in the upstream schema?